<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>725</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>206</red>
        <green>234</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>211</red>
        <green>222</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>117</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>221</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>234</red>
        <green>245</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>230</red>
        <green>244</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>206</red>
        <green>234</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>211</red>
        <green>222</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>117</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>221</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>234</red>
        <green>245</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>230</red>
        <green>244</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>117</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>206</red>
        <green>234</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>211</red>
        <green>222</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>117</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>221</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>117</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>117</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>234</red>
        <green>245</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>234</red>
        <green>245</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>206</red>
        <green>234</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Estudio de Recadación de Lotería - Resultados</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imagenes/dado.png</normaloff>imagenes/dado.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblMedia_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Del Call Center</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>100</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Tiempo usado en la simulación</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTiempo">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtGanVol">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Ganancias promedio (xhr)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtGanCall">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblMedia">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Del voluntariado</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>671</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>30</pointsize>
      <italic>false</italic>
      <underline>true</underline>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Resultados del Análisis</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>1281</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>11</pointsize>
      <italic>true</italic>
      <underline>true</underline>
     </font>
    </property>
    <property name="title">
     <string>Tablas Resultantes de la Simulación</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>50</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Simulación de llamadas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>320</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Simulación de llamadas</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tablaSimLlamadasVol">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>80</y>
       <width>621</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <underline>false</underline>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <column>
      <property name="text">
       <string>Hora</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nro Llamada</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>RND Atendida</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Atendida</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Consumió</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>RND Gasto</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Gasto</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="tablaSimLlamadasCall">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>350</y>
       <width>621</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <underline>false</underline>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <column>
      <property name="text">
       <string>Hora</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nro Llamada</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>RND Atendida</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Atendida</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Consumió</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>RND Gasto</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Gasto</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="tablaSimCall">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>591</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <underline>false</underline>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <column>
      <property name="text">
       <string>Hora</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cantidad Llamadas</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cant. Llamadas Atendidas</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cant. Compras</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ganancia Mujeres</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ganancia Hombres</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ganancia Total</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ganancia Acumulada</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ganancia Promedio</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="tablaSimVolun">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>591</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <underline>false</underline>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <column>
      <property name="text">
       <string>Hora</string>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cantidad Llamadas</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cant. Llamadas Atendidas</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cant. Compras</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ganancia Mujeres</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ganancia Hombres</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ganancia Total</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ganancia Acumulada</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ganancia Promedio</string>
      </property>
     </column>
    </widget>
    <widget class="Line" name="line_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>1251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblSubTitulo_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>15</pointsize>
       <italic>true</italic>
       <underline>true</underline>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Caso VOLUNTARIADO</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSubTitulo_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>290</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>15</pointsize>
       <italic>true</italic>
       <underline>true</underline>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Caso CALL CENTER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>50</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>EN UNA HORA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>320</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>EN UNA HORA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>POR HORA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>320</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>POR HORA</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>lblMedia_2</zorder>
   <zorder>label_2</zorder>
   <zorder>txtTiempo</zorder>
   <zorder>txtGanVol</zorder>
   <zorder>label</zorder>
   <zorder>txtGanCall</zorder>
   <zorder>lblMedia</zorder>
   <zorder>lblTitulo</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
